### PR TITLE
README updates and README template

### PR DESCRIPTION
**Pull request by plusjeff**
Added a README template, but also some small updates to the README file

### CODE REVIEW

File: README.template.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+- `bundle exec rake security` - Run the suite of security scanners on the codebase.
+- `bundle exec rake ci` - Run all build steps performed in Travis CI.
+
+## Deployment Instructions
+
+(TODO: Add stock deployment setup instructions in a future PR)
+
+## How to Contribute
+
+How do people (internal, external, both) contribute to your project? Do they use something like [GitHub Flow](https://guides.github.com/introduction/flow/)? How do people submit bugs? Where does news about the project get released?
```
Comment (ayaleloehr) [inline README.template.md]: Can we create some standard language around this to use as a template?

File: README.template.md [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+
+## Contact
+
+How can folks contact you (the maintainers of the project). GitHub Handles, Email, etc?
```
Comment (ayaleloehr) [inline README.template.md]: Do we have best practice guidelines for our team?

File: README.template.md (hunk 1 above)
Comment (plusjeff) [inline README.template.md]: Will write some up!

File: README.template.md (hunk 2 above)
Comment (plusjeff) [inline README.template.md]: None currently defined, but will take an initial crack at it.

**Comment by plusjeff** [comment]
Added boilerplate text per @ayaleloehr suggestions. @joesoftware : Can you review?

File: README.md [hunk 3]
```diff
@@ -101,5 +90,5 @@ Roadrunner Rails adds some additional gems for making Rails development better.
 - [jshint](https://github.com/damian/jshint) for Javascript.
 
 ### Security
-- [Breakman](https://github.com/presidentbeef/brakeman) for Rails static code analysis for secuirty vulnerabilities
+- [Brakeman](https://github.com/presidentbeef/brakeman) for Rails static code analysis for secuirty vulnerabilities
```
Comment (shanear) [inline README.md]: lol

**Comment by joesoftware** [comment]
The `README.template.md` is still a little sparse but the changes seem to be accurate.
As for `README.md`, the OSX setup stuff looks correct and I tried the steps on my dev machine, but in the future, I think we should add a Linux setup guide as a minimum and consider putting MSWin setup too (the `chocolatey` package manager for MSWin looks pretty mature and could probably be adapted to this README.)

Other than this, LGTM

**Comment by plusjeff** [comment]
Thanks for the review @joesoftware ! Could you open up separate tickets for the Linux and MSWin setup instructions? Hopefully these instructions will be good for Linux folks, but there is likely some differences for Windows users.

**Comment by plusjeff** [comment]
Merging since there was LGTM

**Pull request merged**